lake/utils/wrapper_lake.py

```diff
@@ -4,55 +4,63 @@
 import pytest
 
 from lake.utils.util import *
-from lake.utils.test_infra import base_lake_tester
 from lake.top.lake_top import get_lake_dut
+from lake.top.pond import get_pond_dut
 
 from _kratos import create_wrapper_flatten
 
 
-def get_lake_wrapper(config_path,
-                     stencil_valid,
-                     name,
-                     in_file_name="",
-                     out_file_name="",
-                     in_ports=2,
-                     out_ports=2):
+def wrapper(config_path_input,
```

Comment:
the wrapper function should just take a premade object and wrap it, not decide which constructor to call

Reply:
sure, I can refactor that.

Reply:
Done.